ptf/sainexthop.py

```diff
@@ -0,0 +1,863 @@
+# Copyright 2021-present Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Thrift SAI interface RIF tests
+"""
+from sai_thrift.sai_headers import *
+
+import ptf.mask as mask
+from ptf.packet import *
+from ptf.testutils import *
+from ptf.thriftutils import *
+
+from sai_base_test import *
+
+
+class L3NexthopTest(SaiHelper):
+    '''
+        Basic L3 nexthop tests.
+    '''
+    def runTest(self):
+        self.removeNexthopTest()
+        self.cpuNexthopTest()
+
+    def removeNexthopTest(self):
+        '''
+            Test verifies correct nexthop removal.
```

Comment:
Can you please add some more details as to what the outcome is when nexthop is removed?

Reply:
@pkochmanska  - please address this comment.  thanks.